core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -165,6 +166,19 @@ angular.module('oppia').directive('storyNodeEditor', [
             return '/skill_editor/' + skillId;
           };
 
+          $scope.getPrerequisiteSkillsDescription = function() {
+            let skills = $scope.getPrerequisiteSkillIds();
+            SkillBackendApiService.fetchMultiSkillsAsync(skills).then(
+              function(response) {
+                for (var idx in response) {
+                  $scope.skillIdToSummaryMap[response[idx]._id] =
+                  response[idx]._description;
```

Comment:
Can we use the relevant getters for id and description instead of accessing private properties?

Reply:
Done